bats/index.bats

```diff
@@ -713,7 +713,57 @@ SQL
     [[ "${#lines[@]}" = "1" ]] || false
 }
 
-@test "index: SELECT = Full Match" {
+@test "index: SELECT = Full Match Primary Key" {

```

Comment:
Match primary key? There are tests here for partial match as well

Reply:
I should probably rename this to just `SELECT = Primary Key`. I added the partial part last since it seemed like a good idea later lol